src/python/pose_format/utils/generic.py

```diff
@@ -190,3 +190,28 @@ def reduce_holistic(pose: Pose) -> Pose:
         "FACE_LANDMARKS": face_contours,
         "POSE_LANDMARKS": body_no_face_no_hands
     })
+
+
+def is_left_handed(pose: Pose) -> bool:
+    left_hand = pose.get_components(["LEFT_HAND_LANDMARKS"])
+    right_hand = pose.get_components(["RIGHT_HAND_LANDMARKS"])
+    left_hand_variance = np.nan_to_num(left_hand.body.data).var(axis=0).sum()
+    left_hand_variance = left_hand_variance if left_hand_variance != 'masked' else 0
+    right_hand_variance = np.nan_to_num(right_hand.body.data).var(axis=0).sum()
+    right_hand_variance = right_hand_variance if right_hand_variance != 'masked' else 0
+    return left_hand_variance > right_hand_variance
+
+
+def flip_holistic(pose: Pose) -> Pose:
+    FLIPPED_COMPONENTS = ["POSE_LANDMARKS", "FACE_LANDMARKS", "RIGHT_HAND_LANDMARKS", "LEFT_HAND_LANDMARKS"]
+    FLIPPED_BODY_POINTS = ['NOSE', 'RIGHT_EYE_INNER', 'RIGHT_EYE', 'RIGHT_EYE_OUTER', 'LEFT_EYE_INNER', 'LEFT_EYE', 'LEFT_EYE_OUTER', 'RIGHT_EAR', 'LEFT_EAR', 'MOUTH_RIGHT', 'MOUTH_LEFT', 'RIGHT_SHOULDER', 'LEFT_SHOULDER', 'RIGHT_ELBOW', 'LEFT_ELBOW', 'RIGHT_WRIST', 'LEFT_WRIST', 'RIGHT_PINKY', 'LEFT_PINKY', 'RIGHT_INDEX', 'LEFT_INDEX', 'RIGHT_THUMB', 'LEFT_THUMB', 'RIGHT_HIP', 'LEFT_HIP', 'RIGHT_KNEE', 'LEFT_KNEE', 'RIGHT_ANKLE', 'LEFT_ANKLE', 'RIGHT_HEEL', 'LEFT_HEEL', 'RIGHT_FOOT_INDEX', 'LEFT_FOOT_INDEX']
```

Comment:
import this from https://github.com/sign-language-processing/pose/blob/master/src/python/pose_format/utils/holistic.py#L57